### PR TITLE
Make the plugin localizable

### PR DESCRIPTION
**Pull request by mpfeiffermway**
This updates the plugin with suggestions received from Jenkins CI staff. The Eclipse settings file was removed from the repository and hard coded strings were replaced to allow localization. For testing German localization was added.

### CODE REVIEW

File: src/main/resources/io/relution/jenkins/scmsqs/sqstrigger/help.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div>
+Configure Jenkins to monitor an Amazon SQS queue for messages.  Jenkins will start a new build
+if a message is posted to the queue and the message's content matches this job's repository and
+branch. The repository should be set up to post messages to the queue whenever changes are made.
+<p/>
+Available queues can be configured in Jenkins'
```
Comment (axelhodler) [inline src/main/resources/io/relution/jenkins/scmsqs/sqstrigger/help.html]: whats the `'` at the end for?

File: src/main/resources/io/relution/jenkins/scmsqs/sqstriggerqueue/Messages.properties [hunk 2]
```diff
@@ -0,0 +1,7 @@
+displayName=An Amazon SQS queue configuration
+errorWaitTimeSeconds=Wait Time must be a number between 1 and 20
```
Comment (axelhodler) [inline src/main/resources/io/relution/jenkins/scmsqs/sqstriggerqueue/Messages.properties]: maybe `Waiting period`

File: src/main/resources/io/relution/jenkins/scmsqs/sqstrigger/help.html (hunk 1 above)
Comment (mpfeiffermway) [inline src/main/resources/io/relution/jenkins/scmsqs/sqstrigger/help.html]: Maybe I should've used American spelling an written Jenkins's. It's a possessive of course :)

File: src/main/resources/io/relution/jenkins/scmsqs/sqstriggerqueue/help-maxNumberOfMessages_de.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<div>
+Die maximale Anzahl an Nachrichten die pro Request empfangen werden soll. Amazon SQS liefert
+nie mehr Nachrichten als diesen Wert zurück, kann aber weniger zurückliefern. Zulässige Werte
+gehen von 1 bis 10. Standard sind 10.
+<p/>
+Es werden möglicherweise nicht immer alle Nachrichten zurückgeliefert.
+<p/>
+Größere Werte reduzieren die Anzahl an Requests die gesendet werden müssen wenn mehrere Nachrichten
```
Comment (axelhodler) [inline src/main/resources/io/relution/jenkins/scmsqs/sqstriggerqueue/help-maxNumberOfMessages_de.html]: `,` before wenn? not sure :)

File: src/main/resources/io/relution/jenkins/scmsqs/sqstriggerqueue/Messages.properties (hunk 2 above)
Comment (mpfeiffermway) [inline src/main/resources/io/relution/jenkins/scmsqs/sqstriggerqueue/Messages.properties]: Wait Time is the technical term used by Amazon.

File: src/main/resources/io/relution/jenkins/scmsqs/sqstriggerqueue/help-waitTimeSeconds_de.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<div>
+Zeit, in Sekunden, die auf die Ankunft von Nachrichten in der Queue gewartet werden soll (Long
+Polling). Zulässige Werte gehen von 1 bis 20. Standard sind 20.
+<p/>
+Größere Werte reduzieren die Anzahl an Requests die gesendet werden müssen. Es wird empfohlen den
+maximalen Wert zu verwenden. Reduzieren Sie diesen Wert nur, wenn es zu Problem kommt und Sie sicher
```
Comment (axelhodler) [inline src/main/resources/io/relution/jenkins/scmsqs/sqstriggerqueue/help-waitTimeSeconds_de.html]: `zu Problem kommt` missing plural

**Comment by axelhodler** [comment]
found some minor typos

File: src/main/resources/io/relution/jenkins/scmsqs/sqstriggerqueue/Messages.properties (hunk 2 above)
Comment (axelhodler) [inline src/main/resources/io/relution/jenkins/scmsqs/sqstriggerqueue/Messages.properties]: k :+1:

**Comment by mpfeiffermway** [comment]
Fixed the typos

**Pull request merged**